datastore/graphers/nmdb-graph-network/nmdb-graph-network.cpp

```diff
@@ -161,13 +161,13 @@ class IsRedundantEdge
 // =============================================================================
 // Main code
 // =============================================================================
-
 class Tool : public nmdt::AbstractGraphTool
 {
   private:
     std::map<std::string, Vertex> vertexLookup;
     NetworkGraph graph;
     bool useIcons    {false};
+    bool useIconFolder {false};
```

Comment:
I don't understand what you are using this for.  The icon folder path is not optional.  Either the tool uses the default path or it uses an alternate path.

```suggestion
```